docs/developer-docs/defi/tokens/ledger/usage/icrc1_ledger_usage.mdx

```diff
@@ -0,0 +1,228 @@
+---
+keywords: [intermediate, tokens, icrc, tutorial, ledger]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Using the ICRC-1 ledger
+
+<MarkdownChipRow labels={["Intermediate" , "Tutorial"]} />
+
+## Overview
+There are two main ways to interact with an ICRC-1 ledger.
+- `dfx canister`: the generic canister call from `dfx`.
+- `ic-cdk`: inter-canister calls for the ICRC-1 ledger.
+- [`ledger-icrc-js`](https://github.com/dfinity/ic-js/tree/main/packages/ledger-icrc): a library for interfacing with ICRC-1 ledger on the Internet Computer.
+
+## ICRC-1 and ICRC-1 extension endpoints
+
+Whether your ICRC-1 ledger will have all the endpoints discussed in this tutorial will depend on whether you support any of the extensions of ICRC-1 (ICRC-2, ICRC-3,...).
+This tutorial will go through the endpoints for ICRC-1 and the extension ICRC-2. If you have deployed an ICRC-1 ledger according to the [guide](/docs/developer-docs/defi/tokens/ledger/setup/icrc1_ledger_setup.mdx) you have to make sure that you have enabled ICRC-2 standard. Otherwise these calls will not work.
```

Comment:
```suggestion
This tutorial will go through the endpoints for ICRC-1 and the extension ICRC-2. If you have deployed an ICRC-1 ledger according to the [guide](/docs/developer-docs/defi/tokens/ledger/setup/icrc1_ledger_setup.mdx) you have to make sure that you have enabled the ICRC-2 standard, otherwise, the calls demonstrated in this guide will not work.
```